sim/latency.go

```diff
@@ -20,8 +21,8 @@ func NewLogNormal(seed int64, mean float64) *LogNormalLatency {
 	return &LogNormalLatency{rng: rng, mean: mean}
 }
 
-func (l *LogNormalLatency) Sample() float64 {
+func (l *LogNormalLatency) Sample() time.Duration {
 	norm := l.rng.NormFloat64()
 	lognorm := math.Exp(norm)
-	return lognorm * l.mean
+	return time.Duration(lognorm * l.mean)
```

Comment:
I think this has amounted to changing the units (dividing by nano). The `mean` argument to NewLogNormal should be a duration too, then whatever conversion you make of that to a float needs to be reversed here.

Reply:
Fixed